plugins/outputs/prometheus_client/prometheus_client.go

```diff
@@ -105,6 +106,12 @@ var sampleConfig = `
   ## If set, enable TLS with the given certificate.
   # tls_cert = "/etc/ssl/telegraf.crt"
   # tls_key = "/etc/ssl/telegraf.key"
+  
+  ## If set, enable TLS client authentication with the given CA.
+  # tls_ca = "/etc/ssl/telegraf_ca.crt"
+
+  ## Boolean value indicating whether or not to skip SSL verification
+  # insecure_skip_verify = false
```

Comment:
Remove this option, it's not available for the server configuration.  Don't forget to update the README as well.

Reply:
Done